extensions/src/main/java/org/pentaho/platform/plugin/services/cache/CacheManager.java

```diff
@@ -400,7 +400,10 @@ public Set getAllEntriesFromRegionCache( String region ) {
   public void removeFromRegionCache( String region, Object key ) {
     if ( checkRegionEnabled( region ) ) {
       HvCache hvcache = (HvCache) regionCache.get( region );
-      hvcache.evictEntityData( (String) key );
+      try ( SessionImpl session = (SessionImpl) hvcache.getSessionFactory().openSession() ) {
+        hvcache.getStorageAccess().removeFromCache( key, session );
+        hvcache.evictEntityData( (String) key );
```

Comment:
similar comment for https://github.com/pentaho/pentaho-platform/pull/5511/files#r1471722021

shouldn't we add a log statement with level [error, debug, trace,..] for `MappingException` to class that will handle `hvcache.evictEntityData( (String) key );`

https://github.com/pentaho/pentaho-platform/blob/cefd7023e2cbafcef1c0278bfb4cd8fbfe92bf6c/extensions/src/main/java/org/pentaho/platform/plugin/services/cache/LastModifiedCache.java#L135-L141

The MappingException was being shallowed and there was no way to find this out except for a developer attaching a debugger.

Reply:
I don't believe so.  That exception happens when trying to remove the 2nd level cache which is only used in the carteStatusCache to persist the log xml to a file.  The fact that there is no 2nd level cache persister for these entries is not an error and logging it would just produce spam.

Reply:
The use case would be if we need to customer's environment which we don't have access, but they can give us log files. So would don't have to deploy custom jars and we can't attach debuggers. Customer support or whoever would simply have to adjust logging levels in log4j.

if you find it useful or not address it or not. I'm fine either way.